docs/discover/try-esql.asciidoc

```diff
@@ -75,18 +85,51 @@ FROM kibana_sample_data_logs
 | LIMIT 10
 ----
 +
-. Click **▶Run**.
+. Click **▶Run** again. The table and visualization no longer show results for which the `geo.dest` field value is "GB", and the results are now sorted in descending order in the table based on the `machine.ram` field. 
 +
 [role="screenshot"]
 image:images/esql-full-query.png[An image of the full query result]
 +
 . Click **Save** to save the query and visualization to a dashboard. 
 
-To make changes to the visualization you can use the visualization drop-down. To make changes to the colors used or the axes, or click the pencil icon. This opens an in-line editor where you can change the colors and axes of the visualization. 
+[float]
+==== Edit the ES|QL visualization
```

Comment:
hyper nit: why not have IDs here in case you want to link nicely from another Kibana doc?

Reply:
Because there's a dedicated section for this in the dashboard docs, if a link must exist, it should be to that instead. This is just a bite in the world of [Bite, snack and meal](https://en.wikipedia.org/wiki/Bite,_snack_and_meal)